microsoft/botframework-solutions
# [AI] Documentation Refactor

## Description

- Separate some `docs` folders in `common`, `csharp`, `typescript`
- Add `create a new skill` document
- Update `Virtual Assistant` documentation
- Add a table with the work in progress for the documentation between versions
- Unify the docs betwen `csharp` and `typescript`
- Fix some link references
- Fix tables structure

Close #1034.
Close #941.

## Testing Steps
1. Go to `AI/docs/skills` and `AI/docs/virtual-assistant`.
2. Check the new structure of the documentation.
3. Go to `AI/docs/virtual-assistant/typescript/gettingstarted.md`.
4. Check that the documentation is more verbose now.

## Checklist
